api/src/main/java/org/apache/gravitino/authorization/Privilege.java

```diff
@@ -98,7 +98,13 @@ enum Name {
     /** The privilege to create a tag */
     CREATE_TAG(0L, 1L << 21),
     /** The privilege to apply a tag */
-    APPLY_TAG(0L, 1L << 22);
+    APPLY_TAG(0L, 1L << 22),
+    /** The privilege to create a job template */
+    CREATE_JOB_TEMPLATE(0L, 1L << 23),
+    /** The privilege to use a job template */
+    USE_JOB_TEMPLATE(0L, 1L << 24),
+    /** The privilege to create a job */
+    CREATE_JOB(0L, 1L << 25);
```

Comment:
It is better to align with the job system like "RUN_JOB" instead of "CREATE_JOB"?